### DISCUSSION TITLE
Add tag depending on item id in mods

### DISCUSSION THREAD

**Discussion by Hapseleg** [xivdev/Penumbra#309] [Ideas] [answered]
Hi
When you open up "Changed Items" you can see the different id's of the items/animations/minions etc, we know what those ids mean so why not add a feature where you can see what the changed item is? This helps with at least 2 things:

Gives a better overview in "changed items" 
It can by default add tags, such as gear tags (legs, feet) and animations (dance, joy) and so on.. This also makes it easier to search for mods containing pants for example.

An example of something thats quite messy to look at
![image](https://github.com/xivdev/Penumbra/assets/12915302/f31f1658-a68d-4d81-8e1f-23c8a7e13f2b)

Just an idea for the changed items, would make it a lot easier to get a quick overview
![image](https://github.com/xivdev/Penumbra/assets/12915302/007e004c-df9e-4e5a-ad31-ff1aa2d93dd3)

**Comment by Ottermandias** [comment, marked as answer]
Funnily enough, I started doing this.
I noticed that the equipment piece icons were in a ULD sprite though, and that it would be effort to get them. But I wanted icons instead of text.
So I added a ULD-texture-loader to Dalamud.
But then I forgot about it and didn't do anything with it :D

**Reply by Hapseleg** [reply]
Typical programmer ^^ (also hey, its me from the dalamud discord that you helped ^^)
Did you find something that has all the "item id = item name"?
